src/Installers/Windows/SharedFrameworkBundle/SharedFrameworkBundle.wixproj

```diff
@@ -66,8 +66,8 @@
 
     <BundleNameShort>Microsoft ASP.NET Core $(PackageBrandingVersion)</BundleNameShort>
     <BundleNameSub>Shared Framework</BundleNameSub>
-    <BundleName>$(BundleNameShort) - $(BundleNameSub)</BundleName>
-    <BundleNameFull>$(BundleName) ($(Platform))</BundleNameFull>
+    <BundleName>$(BundleNameShort) - $(BundleNameSub) ($(Platform))</BundleName>
+    <BundleNameFull>$(BundleName)</BundleNameFull>
     <BundleRegName>$(BundleNameFull)</BundleRegName>
```

Comment:
Why keep all of `$(BundleName)`, `$(BundleNameFull)` and `$(BundleRegName)` if their values should be the same❔

More importantly, don't we have the same problem w/ the Windows Hosting bundle https://github.com/dotnet/aspnetcore/blob/52eff90fbcfca39b7eb58baad597df6a99a542b0/src/Installers/Windows/WindowsHostingBundle/WindowsHostingBundle.wixproj#L84-L85❔

Reply:
I guess they correspond to specific wix properties - we could just have one msbuild property that the .wxs file references three times, but I don't think it matters to much. As for the hosting bundle, that's platform-agnostic, so it doesn't need to specify what platform it's for in the name.